localization_launch/launch/localization.launch.xml

```diff
@@ -1,12 +1,17 @@
 <?xml version="1.0"?>
 <launch>
+  <arg name="input/pointcloud" default="/sensing/lidar/top/rectified/pointcloud" description="The topic will be used in the localization util module"/>
```

Comment:
@yukkysaito What is the reason to restore this topic name?
This PR is to changing rectified to outlier.

Reply:
The topic name was not changed this time.
![image](https://user-images.githubusercontent.com/8327598/162192015-124f5e4d-0171-474a-b5ac-d6c6043b468c.png)

Reply:
It seems arguments are deleted…?

Reply:
@h-ohta 
![image](https://user-images.githubusercontent.com/8327598/162250976-f1bac23b-c7c7-4fba-8c96-1b620c8e1dd2.png)

Reply:
I'm sorry I can't understand. Could you explain a bit more by words?

Reply:
sorry, I thought I had written a sentence. :pray: 
It seems input/pointcloud can be used as an argument.

Reply:
Thank you, I see. 
I'm sorry I think my first question makes confuse.

My question is that "why you use /sensing/lidar/top/rectified/pointcloud again?".
This PR title is "change input topic of localization ~ to outlier_filtered/pointcloud". 
So I believe that "/sensing/lidar/top/outlier/pointcloud" can be used here.

Reply:
Or, should we change PR titile?

Reply:
I see. i'll change title

Reply:
@h-ohta done

Reply:
@yukkysaito That PR title is suspicois. The rectified/pointcloud topic has been already used before.
I think This PR is doing `change input pointcloud from outside of launch file`

Reply:
@h-ohta 
I fixed it
If there is anything you think of, you can correct it.